pkgs/development/python-modules/pygatt/default.nix

```diff
@@ -2,51 +2,54 @@
   lib,
   buildPythonPackage,
   fetchFromGitHub,
+  fetchpatch2,
   mock,
-  nose,
   pexpect,
   pyserial,
+  enum-compat,
```

Comment:
This is for Python > 3.4. There was a reason why `pythonOlder` mentioned 3.5 on line 19.

Reply:
https://github.com/NixOS/nixpkgs/pull/330688#discussion_r1698936382

Reply:
We should try to remove the enum-compat dependency, since all Python versions we support have a native enum type.

Reply:
I think it makes sense to just have our `enum-compat` be a stub like our `oldest-supported-numpy`. That way we don’t have to patch or disable dependencies. Indeed, it basically acts like that now, except that it propagates an `enum34` package that is always `null` in practice. So I think this is fine?

Reply:
Just throw it into `pythonRemoveDeps`.